api/graphql/tests/test_reservation_units.py

```diff
@@ -96,8 +96,8 @@ def setUpTestData(cls):
             price_unit=ReservationUnit.PRICE_UNIT_PER_HOUR,
             is_draft=False,
             reservation_start_interval=ReservationUnit.RESERVATION_START_INTERVAL_30_MINUTES,
-            reservation_begins=datetime.datetime.now(),
-            reservation_ends=datetime.datetime.now(),
+            reservation_begins=datetime.datetime.now(tz=get_default_timezone()),
```

Comment:
Django timezone helpers might be a little more clean here. This is not important to change, but up to preference.

[now()](https://docs.djangoproject.com/en/3.2/ref/utils/#django.utils.timezone.now)
[localtime()](https://docs.djangoproject.com/en/4.0/ref/utils/#django.utils.timezone.localtime)

So: `localtime(now())`.